src/nft/components/explore/CarouselCard.tsx

```diff
@@ -228,12 +239,12 @@ const MARKETS_ENUM_TO_NAME = {
   [Markets.LooksRare]: 'LooksRare',
 }
 
-export const CarouselCard = ({ collection, onClick }: CarouselCardProps) => {
+export const CarouselCard = ({ collection, href }: CarouselCardProps) => {
```

Comment:
Is this change documented in the PR desc/title? I think I initially got confused what that had to do with the high-contrast feature!